purchase_order_sending_warning/views/purchase_order.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="purchase_order_view_tree" model="ir.ui.view">
+        <field name="name">purchase.order.view.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_view_tree" />
+        <field name="arch" type="xml">
+            <field name="state" position="after">
+                <field name="sending_warning" readonly="1" />
```

Comment:
to remove since there is already boolean to use, but maybe we want to see it somewhere else?